modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/AccountResourceImpl.java

```diff
@@ -780,6 +783,46 @@ private List<Address> _getContactAddresses(
 			Objects::nonNull);
 	}
 
+	private long _getDefaultBillingAddress(Account account) {
+		long defaultBillingAddressId = GetterUtil.getLong(
+			account.getDefaultBillingAddressId());
+
+		Address billingAddress;
+
+		if (defaultBillingAddressId <= 0) {
```

Comment:
why this can be less then 0?

Reply:
I haven't tried, but can't you provide -14 for instance in the json?